Clarifications to the push module (#2634)
matrix-org/matrix-doc: specification/modules/push.rst

priority. Every push rule MUST have a ``kind`` and ``rule_id``. The ``rule_id``
is a unique string within the kind of rule and its' scope: ``rule_ids`` do not
need to be unique between rules of the same kind on different devices. Rules may
have extra keys depending on the value of ``kind``.

> The different ``kind``\ s of rule in descending order of priority are:

Reviewer: ```suggestion
The different ``kind``\ s of rule, most important first, are:
```
I can never parse "ascending or descending order of priority", for whatever reason

Author: ```suggestion
The different ``kind``\ s of rule, in the order that they are checked, are:
```
